app/views/topics/show.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="container-fluid">
+<h1 class="text-center">Topic:  <%= @topic.title%></h1>
+
+
+
+<div class="row">
+  <% 7.times do |n|%>
```

Comment:
code like this is called "Technical Debt", meaning it's code that we know is problematic and will have to be removed later.  although it's preferable to never introduce technical debt into the codebase, it's also usually unavoidable to be 100% technical debt free.  so, when necessary, at least identify (with a comment) technical debt that will be addressed later.